src/metrics.h

```diff
@@ -0,0 +1,27 @@
+#ifdef HAVE_METRICS
+
+#define METRICS_INCREMENT(field) metrics[field]++;
+
+enum {
+    COUNTER_BOOTP,
+    COUNTER_PXE,
+    COUNTER_DHCPACK,
+    COUNTER_DHCPDECLINE,
+    COUNTER_DHCPDISCOVER,
+    COUNTER_DHCPINFORM,
+    COUNTER_DHCPNAK,
+    COUNTER_DHCPOFFER,
+    COUNTER_DHCPRELEASE,
+    COUNTER_DHCPREQUEST,
+    COUNTER_NOANSWER,
+    COUNTER_MAX,
```

Comment:
should be a `__COUNTER_MAX` better?
It would more obviously that this is no real counter.

Reply:
:+1: